### DISCUSSION TITLE
safetensors

### DISCUSSION THREAD

**Discussion by jclevet** [carson-katri/dream-textures#498] [Ideas]
Hi Carson,
Do you plan on making DREAM TEXTURES compatible with safetensors models?
Thx

**Comment by rknudsen** [comment]
This would be a great feature to include as more and more models are moving to Safetensors format.

**Comment by thelockster101** [comment]
I would love to see this added as I have many .safetensor models I would like to add.
Doesn't help the .ckpt import method doesn't work, I tried converting the .safetensor formats over and no luck as it just didn't add anything.
